package.json

```diff
@@ -157,6 +157,16 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Warn about conflicting extensions and suggest disabling them."
+                },
+                "clangd.inactiveRegions.useOpacity": {
+                    "type": "boolean",
+                    "default": false,
```

Comment:
That is a change to the current behaviour - but I'd prefer the default to be `true`, too.

In any case the new options (and possible change of default behaviour) should be documented in https://github.com/clangd/vscode-clangd/blob/master/CHANGELOG.md (but I don't know how / when this project changes this file).

Reply:
I'm fine with that, I updated the patch to use opacity as the default.

I did invert the naming of the pref (I called it `clangd.inactiveRegions.useBackgroundHighlight` with default value `false`), because it's users who want the non-default (background highlight) behaviour who are more likely to be looking for the pref.

Reply:
> That is a change to the current behaviour

This PR is a change to the current behaviour either way: the previous behaviour was that inactive regions used a different foreground color, now they use a different opacity or a different background color.

> In any case the new options (and possible change of default behaviour) should be documented in https://github.com/clangd/vscode-clangd/blob/master/CHANGELOG.md (but I don't know how / when this project changes this file).

Good idea, thanks.

I added a "Next" section to the changelog, documenting this change and the configuration options. When the next version is actually released, "Next" can be replaced with the version number and release date.